ptf/tests/config/basic_config.py

```diff
@@ -16,16 +17,80 @@ def config_meta_port(self, test):
         """
         Default configuation, with metadata and ports configurations.
         """
+        self.recreate_ports(test)
         self.get_port_list(test)        
         self.get_default_1q_bridge(test)
         self.get_default_vlan(test)
         self.remove_vlan_member(test)
         self.remove_bridge_port(test)
         self.create_host_intf(test)
+        self.create_bridge_port(test)
         self.turn_on_port_admin_state(test)
         #test.set_port_serdes()
 
 
+    def recreate_ports(self, test):
```

Comment:
please check and compare with sairedis recordings, if those RE-create really needed.